orchagent/port.h

```diff
@@ -32,8 +32,9 @@ typedef std::map<sai_vlan_id_t, VlanMemberEntry> vlan_members_t;
 
 struct VlanInfo
 {
-    sai_object_id_t     vlan_oid = 0;
-    sai_vlan_id_t       vlan_id = 0;
+    sai_object_id_t vlan_oid = 0;
+    sai_vlan_id_t vlan_id = 0;
+    sai_vlan_flood_control_type_t vlan_flood_type = SAI_VLAN_FLOOD_CONTROL_TYPE_ALL;
```

Comment:
I suggest to remove this attribute as it is a function handled in intforch and portorch doesn't necessarily be aware of this